### ISSUE TITLE
Duplicating a Matrix block and changing the content results in losing some of this new content when saving the entry

### ISSUE THREAD

**Issue by franciurachel** [verbb/smith#25]
### Describe the bug

We have this Matrix field to display the cast of single performances, and use the Cloning function to duplicate them, changing date and what is needed to avoid having to re-create the shared content.
When cloning a block and changed the content, after we save the entry some of this content is lost (either the image or part of the plain text data).

### Steps to reproduce

1. Create matrix block
2. Clone matrix block
3. Change some of the content
4. Save the entry

Some of the content is lost.

<img width="1016" alt="Screenshot 2022-10-11 at 10 08 58" src="https://user-images.githubusercontent.com/6681670/195036572-670d2ac2-3222-4ca9-bfe8-1d9894160ee4.png">

### Craft CMS version

Craft CMS 3.7.39

### Plugin version

dev-craft-3

### Multi-site?

No

**Comment by engram-design** [comment]
So this includes nested Super Table content. Are you changing the content of just the Matrix blocks, or the inner Super Table blocks as well?

**Comment by franciurachel** [comment]
@engram-design  Yes, it's a Matrix and each block has a Date field and 2 Supertable fields. We are changing the content of all of them, so Yes, also of the inner Super Table blocks

**Comment by engram-design** [comment]
Hmm, I'm wondering if you might be able to put together a video of you recreating the issue? I've recreated your field, and unable to replicate it.

Here's a quick video of my setup, in case something is different https://share.cleanshot.com/jitmya
